src/main/res/values/strings.xml

```diff
@@ -3,14 +3,14 @@
     <string name="action_failed">Action failed</string>
     <string name="add_repository">Add repository</string>
     <string name="address">Address</string>
-    <string name="all_applications">All Applications</string>
-    <string name="all_applications_up_to_date">All Applications up to date</string>
+    <string name="all_applications">All applications</string>
+    <string name="all_applications_up_to_date">All your applications are up to date</string>
     <string name="already_exists">Already exists</string>
     <string name="always">Always</string>
     <string name="amoled">Amoled</string>
```

Comment:
```suggestion
    <string name="amoled">Black</string>
```
Should be AMOLED, but more correct to call it "Black" if it is the theme setting.